angular-workspace/projects/ni/nimble-angular/src/directives/anchor-menu-item/nimble-anchor-menu-item.directive.ts

```diff
@@ -0,0 +1,79 @@
+import { Directive, ElementRef, Input, Renderer2 } from '@angular/core';
+import type { AnchorMenuItem } from '@ni/nimble-components/dist/esm/anchor-menu-item';
+import { BooleanValueOrAttribute, toBooleanProperty } from '../utilities/template-value-helpers';
+
+export type { AnchorMenuItem };
+
+/**
+ * Directive to provide Angular integration for the anchor menu item.
+ */
+@Directive({
+    selector: 'nimble-anchor-menu-item'
+})
+export class NimbleAnchorMenuItemDirective {
```

Comment:
Why can't this extend `NimbleAnchorBaseDirective`?

Reply:
Great point. I totally forgot about the base directive.